Gemfile

```diff
@@ -113,6 +113,9 @@ group :development, :test do
   gem 'i18n_generators'
 
   gem 'sprockets', '~>2.12.3' # spckets 3.0.3 breaks konacha
+
+  # Mocks and stubs for javascript tests
+  gem 'sinon-rails'
```

Comment:
Unfortunately, this gem version is outdated and no longer maintained.

Current Sinon is 1.15.0 but the gem provides 1.10.3.

What do you think about putting sinon.js on the `vendor` directory?